certbot-ci/certbot_integration_tests/certbot_tests/test_main.py

```diff
@@ -657,4 +732,4 @@ def test_preferred_chain(context):
 
         with open(conf_path, 'r') as f:
             assert 'preferred_chain = {}'.format(requested) in f.read(), \
-                   'Expected preferred_chain to be set in renewal config'
\ No newline at end of file
+                   'Expected preferred_chain to be set in renewal config'
```

Comment:
I think this was the result of my editor.

Reply:
What do you mean here?

Reply:
When I last modified this file, I failed to include a trailing newline. @atombrella's editor appears to automatically add them.

Reply:
Yes. Exactly. #8297 is an attempt to address some of these settings.